<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>674</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <attribute name="title">
           <string>终端</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QTextEdit" name="receive_content">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>300</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Fixedsys</family>
              </font>
             </property>
             <property name="acceptDrops">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">border:0px;
background-color:#000;
color:#FFF;
</string>
             </property>
             <property name="undoRedoEnabled">
              <bool>false</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="sendcontent">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Fixedsys</family>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:#000;
color:#FFF;</string>
             </property>
             <property name="frame">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>高级设置</string>
          </attribute>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>70</y>
             <width>431</width>
             <height>221</height>
            </rect>
           </property>
           <property name="title">
            <string>事件触发器</string>
           </property>
           <widget class="QPushButton" name="new_trigger">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>30</y>
              <width>161</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>新建事件(字符，时间)</string>
            </property>
           </widget>
           <widget class="QListView" name="trigger_list">
            <property name="geometry">
             <rect>
              <x>25</x>
              <y>61</y>
              <width>381</width>
              <height>151</height>
             </rect>
            </property>
           </widget>
          </widget>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>30</y>
             <width>161</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Feature To done...</string>
           </property>
          </widget>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>330</y>
             <width>431</width>
             <height>301</height>
            </rect>
           </property>
           <property name="title">
            <string>QC自动测试</string>
           </property>
           <widget class="QPlainTextEdit" name="qc_test_log">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>110</y>
              <width>371</width>
              <height>181</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_6">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>90</y>
              <width>54</width>
              <height>12</height>
             </rect>
            </property>
            <property name="text">
             <string>测试日志</string>
            </property>
           </widget>
           <widget class="QPushButton" name="qc_open">
            <property name="geometry">
             <rect>
              <x>240</x>
              <y>50</y>
              <width>75</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>打开..</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="qc_file_path">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>50</y>
              <width>201</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
           <widget class="QPushButton" name="qc_start_test">
            <property name="geometry">
             <rect>
              <x>320</x>
              <y>50</y>
              <width>75</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>开始测试</string>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="title">
          <string>设置</string>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>13</x>
            <y>25</y>
            <width>24</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>串口</string>
          </property>
         </widget>
         <widget class="QComboBox" name="setting_port">
          <property name="geometry">
           <rect>
            <x>58</x>
            <y>25</y>
            <width>121</width>
            <height>22</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Fixedsys</family>
           </font>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>13</x>
            <y>53</y>
            <width>36</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>波特率</string>
          </property>
         </widget>
         <widget class="QComboBox" name="setting_baud">
          <property name="geometry">
           <rect>
            <x>58</x>
            <y>53</y>
            <width>121</width>
            <height>22</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Fixedsys</family>
           </font>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>13</x>
            <y>81</y>
            <width>36</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>数据位</string>
          </property>
         </widget>
         <widget class="QComboBox" name="setting_databits">
          <property name="geometry">
           <rect>
            <x>58</x>
            <y>81</y>
            <width>121</width>
            <height>22</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Fixedsys</family>
           </font>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>13</x>
            <y>109</y>
            <width>36</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>停止位</string>
          </property>
         </widget>
         <widget class="QComboBox" name="setting_stopbit">
          <property name="geometry">
           <rect>
            <x>58</x>
            <y>109</y>
            <width>121</width>
            <height>22</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Fixedsys</family>
           </font>
          </property>
         </widget>
         <widget class="QPushButton" name="open">
          <property name="geometry">
           <rect>
            <x>13</x>
            <y>140</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>打开</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QLabel" name="status">
          <property name="geometry">
           <rect>
            <x>13</x>
            <y>166</y>
            <width>161</width>
            <height>16</height>
           </rect>
          </property>
          <property name="palette">
           <palette>
            <active>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>118</red>
                <green>116</green>
                <blue>108</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="font">
           <font>
            <family>Fixedsys</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>status</string>
          </property>
         </widget>
         <widget class="QPushButton" name="clear">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>140</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>清空</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="history_list">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>999999</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>150</height>
          </size>
         </property>
         <property name="title">
          <string>日志</string>
         </property>
         <widget class="QPushButton" name="startrecord">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>40</y>
            <width>131</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>开始记录</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="open_dir">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>110</y>
            <width>131</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>打开目录</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QCheckBox" name="time_check">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>80</y>
            <width>131</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>加入时间前缀</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <zorder></zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
